panther_analysis_tool/main.py

```diff
@@ -398,6 +399,95 @@ def upload_analysis(args: argparse.Namespace) -> Tuple[int, str]:
     return 0, ""
 
 
+def confirm_analysis_exists(args: argparse.Namespace, analysis_id_list: list) -> list:
+    validation_payload = {"listDetections": {"ids": analysis_id_list, "fields": ["id"]}}
+    client = get_client(args.aws_profile, "lambda")
+
+    validation = client.invoke(
+        FunctionName="panther-analysis-api",
+        InvocationType="RequestResponse",
+        LogType="None",
+        Payload=json.dumps(validation_payload),
+    )
+
+    validation_string = json.loads(validation["Payload"].read().decode("utf-8"))["body"]
+    # To handle mocks in testing, converting to string does not result in a processable string
+    if isinstance(validation_string, dict):
```

Comment:
I really am not a fan of this, but in testing the validation string is returned as a dict. Type casting it to a string results in a string that can not be parsed by the json module because of a the entities are not in double quotes, attempting to replace the single quotes with double quotes results in invalid json.  

The machinations of trying to manipulate it back into a processesable string where getting pretty ugly ( and still not working), and this seemed like a more simple solution rather than deep diving on what was a happening in the mock. I am absolutely open to other ideas if anyone has them.

cc: @wey-chiang  This is what I wanted to talk about at 3